checker-qual/src/main/java/org/checkerframework/checker/mustcall/qual/CreatesObligation.java

```diff
@@ -0,0 +1,64 @@
+package org.checkerframework.checker.mustcall.qual;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Repeatable;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import org.checkerframework.framework.qual.InheritedAnnotation;
+import org.checkerframework.framework.qual.JavaExpression;
+
+/**
+ * Indicates that the method resets the must-call obligation on the target (the value argument) to
+ * its declaration. This effectively undoes flow-sensitive type refinement.
+ *
```

Comment:
I have done so, but as I've said before it doesn't make much sense without the context of the full consistency checker. The example will be useful later when we merge that, but for now it's somewhat misleading - it describes features that aren't part of this PR.

Since `@CreatesObligation` is an undocumented feature, I think that's fine for now.